dart/tool/common/generate_common.dart

```diff
@@ -8,11 +8,22 @@ import 'package:markdown/markdown.dart';
 
 import 'src_gen_common.dart';
 
-/// [ApiParseUtil] contains top level parsing utilities
+/// [ApiParseUtil] contains top level parsing utilities.
 class ApiParseUtil {
-  /// Extract the current VM Service version number
-  List<int> parseServiceVersion(List<Node> nodes) {
+  /// Extract the current VM Service version number as a String.
+  static String parseVersionString(List<Node> nodes) {
+    final RegExp regex = new RegExp(r'[\d.]+');
```

Comment:
Will this RegExp match a single period at the end of a sentence? Would it be safer to be more specific?